db/seeds.rb

```diff
@@ -5,3 +5,10 @@
 #
 #   movies = Movie.create([{ name: 'Star Wars' }, { name: 'Lord of the Rings' }])
 #   Character.create(name: 'Luke', movie: movies.first)
+Train.delete_all
```

Comment:
fyi, there's a `find_or_create_by` method you can use to basically say "make one of these things if it doesn't already exist but don't duplicate it" 🙂but no changes required here